app/views/todo_mailer/weekly_report.html.erb

```diff
@@ -1,12 +1,22 @@
-<h1>Hi, <%= @user.email %></h1>
+<h1>Hi, <%= @user.nombre %></h1>
 
 <p>
   Yout weekly report from ToDo App 
 </p>
-
 <h3>Your ToDo's</h3>
-<%# @todo.each do |todo|%>
-<ul>
-<%= @todo.description %>
-</ul>
-<% #end %>
\ No newline at end of file
+<table>
+<tr>
+<th>Description</th>
+<th>Status</th>
+</tr>
+<% @user.todos.each do |todo| %>
+<tr>
+  <td>
+  <%= todo.description %>
+  </td>
+  <td>
+  <%= todo.estate?  ? "In process" : "Finish" %>
+  </td>
+</tr>
+<% end %>
+</table>
```

Comment:
Fix your indentation across this file, and add a new line at the end of the file, please.